src/include/sof/trace.h

```diff
@@ -212,7 +206,7 @@ void trace_flush(void);
 void trace_off(void);
 void trace_init(struct sof *sof);
 
-#if TRACE
+#if defined(CONFIG_TRACE)
```

Comment:
Please use `#if CONFIG_*`, ifdefs break when value evaluate to false

Reply:
I've changed.